polymer.json

```diff
@@ -1,6 +1,9 @@
 {
 	"npm": true,
 	"lint": {
-		"rules": ["polymer-3"]
+		"rules": ["polymer-3"],
+		"ignoreWarnings": [
+			"non-literal-import"
```

Comment:
Why this rule?

Reply:
Otherwise Polymer linting complains when you import a dynamically named file, such as `import(importUri)` - build won't be able to analyze what to include in bundles etc.